controllers/v1/userProjects.js

```diff
@@ -434,11 +432,11 @@ module.exports = class UserProjects extends Abstract {
     }
```

Comment:
@Prajwal17Tunerlabs do we require req.userDetails.userToken in solutionDetails api. Are we calling any services internally there